app/controllers/renewing_registrations_controller.rb

```diff
@@ -4,9 +4,15 @@ class RenewingRegistrationsController < ApplicationController
   before_action :authenticate_user!
 
   def show
-    transient_registration = WasteCarriersEngine::RenewingRegistration.find_by(reg_identifier: params[:reg_identifier])
+    begin
+      transient_registration = WasteCarriersEngine::RenewingRegistration.find_by reg_identifier: params[:reg_identifier]
```

Comment:
```suggestion
      transient_registration = WasteCarriersEngine::RenewingRegistration.find_by(reg_identifier: params[:reg_identifier])
```

Can we put brackets around this for consistency? 🙏  I know it makes the line too long and annoys Rubocop but I'd rather find another way around it.

Reply:
Sure :) 

_she hates rubocop_